packages/extension/src/view/devtools/components/cookies/cookieLanding/framesSection.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import React, { useMemo } from 'react';
+import {
+  CookiesLandingWrapper,
+  MatrixContainer,
+  prepareFrameStatsComponent,
+  type MatrixComponentProps,
+  LEGEND_DESCRIPTION,
+} from '@ps-analysis-tool/design-system';
+import {
+  ORPHANED_COOKIE_KEY,
+  UNMAPPED_COOKIE_KEY,
+} from '@ps-analysis-tool/common';
```

Comment:
I think @amovar18 worked on removing the need for orphaned and unmapped cookies. @amovar18 has that work been merged to develop?